packages/doenetml-worker-javascript/src/utils/externalPathResolution.ts

```diff
@@ -0,0 +1,278 @@
+import {
```

Comment:
This is only used for testing, right? Do we have a place to put testing-only functions? If not, the file/functions should be clearly labeled, but my guess is that you can move this directly to the `test/` folder. If it does not end in `.test.ts` it will be ignored by vitest.

Reply:
This function is used not just for testing but also for debug functions from the production server. Eventually, this should be moved into `debug-hooks` workspace or something similar. But, that ended up being a bigger task than seems reasonable to do at the moment, given that we have to deal with a web worker in some contexts and not in others. For now, I'm leaving it here but with a TODO to eventually move it.